UIDokan/Sources/UIDokan/Extentions/Colors+Style.swift

```diff
@@ -1,3 +1,12 @@
+import UIKit.UIColor
```

Comment:
Let's add a comment here to indicate that this is a generated file to avoid any manual update to it.. or we can rename the result file to `UIColors.Generated.swift`

Reply:
Done